droidlet/dashboard/web/src/StateManager.js

```diff
@@ -67,13 +68,15 @@ class StateManager {
     timelineDetails: [],
     timelineFilters: ["Perceive", "Dialogue", "Interpreter", "Memory"],
     timelineSearchPattern: "",
+    agentType: "locobot",
   };
   session_id = null;
 
   constructor() {
     this.processMemoryState = this.processMemoryState.bind(this);
     this.setChatResponse = this.setChatResponse.bind(this);
     this.setConnected = this.setConnected.bind(this);
+    this.updateAgentState = this.updateAgentState.bind(this);
```

Comment:
nit: updateAgentType is better than updateAgentState imho, I'll leave it to you to make the call.

Reply:
updated